ztrouy/frontend-capstone-api
# Feature - Genres and Platforms

### Purpose
To enable Users to see the Genres and Platforms available on Game Gauge

### Files Changed
- Created file, and wrote initial Serializer and View Set for Genres in new `genres.py`
- Created file, and wrote initial Serializer and View Set for Platforms in new `platforms.py`
- Imported Genre and Platform View Sets in `__init__.py` package in `/views` directory
- Added routing for Genre and Platform View Sets in `urls.py`

### To Test
Fetch, setup using `pipenv shell` and `./seed_database.sh` in project root directory, run the project via the debugger, then confirm the following
- GET endpoint `/genres` returns a list of all Genres with their Id and Name
- GET endpoint `/genres/{id}` returns the appropriate Genre if it exists
- GET endpoint `/platforms` returns a list of all Platforms with their Id and Name
- GET endpoint `/platforms/{id}` returns the appropriate Platform if it exists

closes #6 
closes #7
